Changelog.md

```diff
@@ -1,5 +1,12 @@
 ## [Unreleased]
 
+- PR#164 (no sha yet)
```

Comment:
I guess we should first clarify what the purpose of the changelog is. I'm mostly following http://keepachangelog.com/en/0.3.0/

I don't think we care about the PR or the SHA. Things like "...doesn't hide exceptions..." also seem like too much detail for how I think about this.

Reply:
Fixed.